packages/turf-center-of-mass/README.md

```diff
@@ -9,7 +9,8 @@ using this formula: [Centroid of Polygon](https://en.wikipedia.org/wiki/Centroid
 
 **Parameters**
 
--   `feature` **[Feature](http://geojson.org/geojson-spec.html#feature-objects)** the feature
```

Comment:
Are these changes hand-written? The readmes are auto-generated with the generate-readmes script https://github.com/Turfjs/turf/blob/master/scripts/generate-readmes Running `./scripts/generate-readmes` may overwrite these cahnges